test/e2e/launch.js

```diff
@@ -43,12 +44,13 @@ function launch(t) {
       })
 
     started = new Promise(async (resolve, reject) => {
-      console.log("using binary", binary)
+      console.log("using cli binary", binary)
```

Comment:
is this always going to refer to `gaiacli`?

Reply:
Yes, but the path to `gaiacli`